### PR TITLE
Upgraded Phase to v0.1.2-7

### PR DESCRIPTION
**Pull request by kinyipchan913**
Access state completeness
Add python wrapper functions

### CODE REVIEW

**Comment by benknight135** [comment]
@kinyipchan913 You need update the Phase version number in the github workflows

[.github/workflows/pull_request.yml, line 9](https://github.com/i3drobotics/pyphase/blob/fe46f28fc5c65f76774046cbceae3000fdfa368d/.github/workflows/pull_request.yml#L9)
```
PHASE_VERSION: v0.0.27
```

[.github/workflows/build.yml, line 10](https://github.com/i3drobotics/pyphase/blob/fe46f28fc5c65f76774046cbceae3000fdfa368d/.github/workflows/build.yml#L10)
```
PHASE_VERSION: v0.0.27
```

[.github/workflows/release.yml, line 16](https://github.com/i3drobotics/pyphase/blob/fe46f28fc5c65f76774046cbceae3000fdfa368d/.github/workflows/release.yml#L16)
```
PHASE_VERSION: v0.0.27
```

Also update reference in [gitpod.Dockerfile](https://github.com/i3drobotics/pyphase/blob/fe46f28fc5c65f76774046cbceae3000fdfa368d/.gitpod.Dockerfile#L3)
```
RUN curl --output phase.deb -L https://github.com/i3drobotics/phase/releases/download/v0.0.27/phase-v0.0.27-ubuntu-20.04-x86_64.deb && \
```

And in multiple references in README's: [README.md](https://github.com/i3drobotics/pyphase/blob/fe46f28fc5c65f76774046cbceae3000fdfa368d/README.md?plain=1#L33) and [docs/README_developer.md](https://github.com/i3drobotics/pyphase/blob/fe46f28fc5c65f76774046cbceae3000fdfa368d/docs/README_developer.md?plain=1#L10)
```
Download debian package from [v0.0.27 release](https://github.com/i3drobotics/phase/releases/tag/v0.0.27).  
```

File: src/bindings/calib/cameracalibration_bindings.cpp [hunk 1]
```diff
@@ -81,17 +81,15 @@ void init_cameracalibration(py::module_ &m) {
             translation_y : float
                 Translation in y axis of calibration file
             )")
-        //.def("rectify", &I3DR::Phase::CameraCalibration::rectify)
-        .def("remapPoint", &I3DR::Phase::CameraCalibration::remapPoint, R"(
-            Remap point
+        .def("rectify", &I3DR::Phase::CameraCalibration::rectify, R"(
```
Comment (benknight135) [inline src/bindings/calib/cameracalibration_bindings.cpp:84]: In the C++ Phase the rect_image input is filled with the rectifed image. This is passed as reference so I wasn't sure if this would work with the numpy conversions. Can you manually check that the rect_image is updated when input to this function?

File: test/src/calib/test_cameracalibration.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
  @details Unit tests generated using PyTest
```
Comment (benknight135) [inline test/src/calib/test_cameracalibration.py:9]: performance testing is not part of this PR. Should remove and include this in seperate PR that looks specifically at performance testing

File: src/bindings/calib/stereocalibration_bindings.cpp [hunk 3]
```diff
@@ -18,6 +18,14 @@ namespace py = pybind11;
...
+    py::enum_<I3DR::Phase::CalibrationSelection>(m, "CalibrationSelection", R"(
+        Structure of CalibrationSelection
+
+        )")
+        .value("LEFT", I3DR::Phase::CalibrationSelection::LEFT)
+        .value("RIGHT", I3DR::Phase::CalibrationSelection::RIGHT)
+        .export_values();
+    
     py::class_<I3DR::Phase::StereoCameraCalibration>(m, "StereoCameraCalibration", R"(
             Variables of stereo camera calibration file
```
Comment (benknight135) [inline src/bindings/calib/stereocalibration_bindings.cpp]: I think "Variables of stereo camera calibration file" should be changed to "Store and manipulate stereo camera calibration data"

File: src/bindings/calib/cameracalibration_bindings.cpp (hunk 1 above)
Comment (benknight135) [inline src/bindings/calib/cameracalibration_bindings.cpp:84]: In the C++ Phase the rect_image input is filled with the rectifed image. This is passed as reference so I wasn't sure if this would work with the numpy conversions. Can you manually check that the rect_image is updated when input to this function?

**Review by benknight135 — CHANGES_REQUESTED** [review]

File: src/bindings/calib/stereocalibration_bindings.cpp (hunk 3 above)
Comment (kinyipchan913) [reply src/bindings/calib/stereocalibration_bindings.cpp]: Done

File: test/src/calib/test_cameracalibration.py (hunk 2 above)
Comment (kinyipchan913) [reply test/src/calib/test_cameracalibration.py:9]: Done
